src/BorrowLogic/BorrowHandlers.sol

```diff
@@ -57,39 +47,7 @@ abstract contract BorrowHandlers is IBorrowHandlers, BorrowCheckers, SafeMint {
             );
         }
 
-        // the shortest duration offered among all offers used will be used to determine the loan end date.
-        if (arg.offer.duration < collatState.minOfferDuration) {
-            collatState.minOfferDuration = arg.offer.duration;
-        }
-        if (arg.offer.loanToValue < collatState.minOfferLoanToValue) {
-            collatState.minOfferLoanToValue = arg.offer.loanToValue;
-        }
-        if (arg.offer.loanToValue > collatState.maxOfferLoanToValue) {
-            collatState.maxOfferLoanToValue = arg.offer.loanToValue;
-        }
-
-        /* This check serves to prevent a manipulation of the auction start price. The auction price is determined by
-        a multiple of the mean of the offer loanToValues used in the loan. Being lender and borrower at the same time,
-        one could influence this price by providing an infinitesimal loanToValue price, get a loan instantly liquidable
-        at a price inferior a the loanToValue of another offer used in the loan. buying its own NFT in auction would
-        result in a net gain arising from the difference between the sale price of the NFT and the loanToValue provided
-        by the other offer (effectively stealing the funds of the lender). This is prevented by limiting the max gap
-        between two offer loanToValue to a factor 2, and making the priceFactor of the auction equal or superior to 2.5.
-        In the worst case, the attacker can influence the start price of the auction to be
-        (attacked_offer_ltv / 2) * 2.5 which is superior to the attacked offer loan to value. */
-        if (collatState.maxOfferLoanToValue > collatState.minOfferLoanToValue * 2) {
-            revert UnsafeOfferLoanToValuesGap(
-                collatState.minOfferLoanToValue,
-                collatState.maxOfferLoanToValue
-            );
-        }
-
-        // transferring the borrowed funds from the lender to the borrower
-        collatState.assetLent.checkedTransferFrom(signer, collatState.from, arg.amount);
-
-        // issuing supply position NFT to the signer of the loan offer with metadatas
-        safeMint(signer, Provision({amount: arg.amount, share: shareMatched, loanId: collatState.loanId}));
-        return (collatState);
+        return (collatState, signer);
```

Comment:
Checks are no longer relevant. Mint is removed but this is fine because provisions are minted by useCollateral instead